src/dxvk/hud/shaders/hud_graph_frag.frag

```diff
@@ -39,7 +40,7 @@ void main() {
 
   float value = mix(p0.value, p1.value, fx);
   float alpha = value + v_coord.y - 1.0f;
-  alpha = min(alpha / dFdy(v_coord.y), 1.0f);
+  alpha = min(alpha / dFdy(v_coord.y), opacity);
```

Comment:
Shouldn't we be multiplying the final alpha value with `opacity`?

Reply:
I'm rather new to shaders, but I've observed it has the same end effect here. If it helps with clarity, I'll do the multiplication at the end, sure.